EIPS/eip-1753.md

```diff
@@ -0,0 +1,249 @@
+---
+eip: 1753
+title: Smart Contract Interface for Licences
+author: Lucas Cullen (@BitcoinBrisbane), Kai Yeung (@CivicKai), Anna Crowley <annaelizabethcrowley@gmail.com>, Caroline Marshall <caroline.marshall888@gmail.com>, Katrina Donaghy, <katrina@civicledger.com>
```

Comment:
There's a typo here, the comma before the email address.

Reply:
Good spot, fixed.